src/osd/PG.cc

```diff
@@ -1571,8 +1563,14 @@ void PG::on_role_change() {
 }
 
 void PG::on_new_interval() {
-  dout(20) << __func__ << " scrub_queued was " << scrub_queued << " flags: " << m_planned_scrub << dendl;
-  scrub_queued = false;
+  // log some scrub data before we react to the interval
+  if (m_scrubber) {
+    dout(20) << __func__ << (m_scrubber->is_being_scrubbed() ? " scrubbing " : " ")
+             << "flags: " << m_planned_scrub << dendl;
+  } else {
+    dout(20) << __func__ << dendl;
```

Comment:
do we need this, can this not be inferred?

Reply:
I do wish to know when a new interval starts but the PG has no Scrubber object. Added as I needed that
during debugging.